spot_rl_experiments/spot_rl/real_policy.py

```diff
@@ -296,7 +412,7 @@ def act(self, observations, expert=None):
 
 if __name__ == "__main__":
     gaze_policy = GazePolicy(
-        "weights/bbox_mask_5thresh_autograsp_shortrange_seed1_36.pth",
+        "weights/final_paper/gaze_normal_32_seed100_1649708902_ckpt.38.pth",
```

Comment:
This should probably also come from a config or CLI

Reply:
changed to load from config @zephirefaith , will reflect in next commit